.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,21 @@
+Reviewers: GitHub @<name>s of all reviewers
+
+Linked PRs: <links to corresponding PRs, optional>
+
+## Changes
+- <changes here>
+
+## TODOs:
+- [ ] +1
+- [ ] Updated README
+- [ ] Updated CHANGELOG
+- [ ] <Other applicable TODOs>
+- [ ] <Feedback from reviewers>
+
+## How To Test
+- <necessary config changes>
```

Comment:
Since this template is for this code style repo... what about being a bit more specific in this section?
```
* Linting rules work as expected (if you made rule changes that affect linting)
* ??
```